samples/excel/89-preview-apis/chart-axis.yaml

```diff
@@ -6,7 +6,7 @@ host: EXCEL
    // Set vertical axis title.\n        valueAxis.title.text = \"Number of items\";\n        \n        // Show small gridlines.\n        valueAxis.minorGridlines.visible = true;\n\n        categoryAxis.title.load(\"text\");\n        valueAxis.load(\"text\");\n        valueAxis.minorGridlines.load(\"visible\");\n\n        await context.sync();\n\n        console.log(\"The category axis title is: \" + categoryAxis.title.text);\n        console.log(\"The value axis title is: \" + valueAxis.title.text);\n        \n        OfficeHelpers.UI.notify(\"The minor gridlines visibility is set to: \" + valueAxis.minorGridlines.visible);\n    });\n}\n\nasync function setup() {\n    await Excel.run(async (context) => {\n        const sheet = await OfficeHelpers.ExcelUtilities\n            .forceCreateSheet(context.workbook, \"Sample\");\n        let expensesTable = sheet.tables.add('A1:E1', true);\n        expensesTable.name = \"SalesTable\";\n\n        expensesTable.getHeaderRowRange().values = [[\"Product\", \"Qtr1\", \"Qtr2\", \"Qtr3\", \"Qtr4\"]];\n\n        expensesTable.rows.add(null, [\n            [\"Frames\", 5000, 7000, 6544, 4377],\n            [\"Saddles\", 400, 323, 276, 651],\n            [\"Brake levers\", 12000, 8766, 8456, 9812],\n            [\"Chains\", 1550, 1088, 692, 853],\n            [\"Mirrors\", 225, 600, 923, 544],\n            [\"Spokes\", 6005, 7634, 4589, 8765]\n        ]);\n\n        if (Office.context.requirements.isSetSupported(\"ExcelApi\", 1.7)) {\n            sheet.getUsedRange().format.autofitColumns();\n            sheet.getUsedRange().format.autofitRows();\n        }\n\n        createChart(context);\n        addVerticalAxisLabel(context);\n        sheet.activate();\n\n        await context.sync();\n    });\n}\n\nasync function createChart(context: Excel.RequestContext) {\n    const sheet = context.workbook.worksheets.getItem(\"Sample\");\n    const salesTable = sheet.tables.getItem(\"SalesTable\");\n\n    const dataRange = salesTable.getRange();\n\n    let chart = sheet.charts.add(\"ColumnClustered\", dataRange, Excel.ChartSeriesBy.columns);\n\n    chart.setPosition(\"A15\", \"F30\");\n    chart.title.text = \"Quarterly sales chart\";\n    chart.legend.position = \"right\"\n    chart.legend.format.fill.setSolidColor(\"white\");\n    chart.dataLabels.format.font.size = 15;\n    chart.dataLabels.format.font.color = \"black\";\n    let points = chart.series.getItemAt(0).points;\n}\n\nasync function addVerticalAxisLabel(context: Excel.RequestContext) {\n\n        const sheet = context.workbook.worksheets.getItem(\"Sample\");\n\n        let chart = sheet.charts.getItemAt(0);\n        let axis = chart.axes.valueAxis;\n        axis.displayUnit = \"Thousands\";\n}\n\n/** Default helper for invoking an action and handling errors. */\nasync function tryCatch(callback) {\n    try {\n        await callback();\n    }\n    catch (error) {\n        OfficeHelpers.UI.notify(error);\n        OfficeHelpers.Utilities.log(error);\n    }\n}\n"
```

Comment:
Remove the "async" keyword from the createChart and addVerticalAxisLabel functions. They don't have anything awaitable in them.